main.cpp

```diff
@@ -119,7 +144,7 @@ FloatType& FloatType::multiply ( const float y)
 
 FloatType& FloatType::divide ( const float y ) 
 { 
-    if (abs(y) > 0) 
+    if (std::abs(y) > 0) 
```

Comment:
Floating point division by zero is not an error.  Integer division by zero is an error and will crash the program.
- [ ] only warn.

Reply:
Because the compiler warns me not to compare a float value to 0 , I have opted to do it this way with a logical not rather than `if {} else {}`

`FloatType& FloatType::divide ( const float y ) 
{ 
    *pointerToFloatValue /= y;
    if ( !(std::abs(y) > 0) ) 
    {
        std::cout <<    "\x1B[31m Divide-by-zero warning! \x1B[0m" << std::endl; 
    }   
    return *this;      
}`